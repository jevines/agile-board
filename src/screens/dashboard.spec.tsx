import React from "react";
import { render } from "@testing-library/react";
import Dashboard from "./dashboard";

describe("The Dashboard Component", () => {
  it("should match the snapshot", () => {
    const { container } = renderDashboard();
    expect(container.firstChild).toMatchSnapshot();
  });
});

const renderDashboard = () => {
  return render(<Dashboard />);
};
