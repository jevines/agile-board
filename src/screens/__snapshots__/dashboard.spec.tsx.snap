// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Dashboard Component should match the snapshot 1`] = `
<section
  class="sc-gKsewC jJxQUn"
>
  <section
    class="sc-eCssSg foUHDZ"
  >
    <h1
      class="sc-jSgupP dhcVWK"
    >
      To Do
    </h1>
    <section
      class="sc-bdfBwQ bCQgUO"
    >
      <h2
        class="sc-gsTCUz gzfVJk"
      >
        mock task title
      </h2>
      <p
        class="sc-dlfnbm bglaea"
      >
        A mock description
      </p>
      <p
        class="sc-hKgILt fLjEpC"
      >
        comment 1
      </p>
      <p
        class="sc-hKgILt fLjEpC"
      >
        comment 2
      </p>
    </section>
  </section>
  <section
    class="sc-eCssSg foUHDZ"
  >
    <h1
      class="sc-jSgupP dhcVWK"
    >
      In Progress
    </h1>
    <section
      class="sc-bdfBwQ bCQgUO"
    >
      <h2
        class="sc-gsTCUz gzfVJk"
      >
        mock task title
      </h2>
      <p
        class="sc-dlfnbm bglaea"
      >
        A mock description
      </p>
      <p
        class="sc-hKgILt fLjEpC"
      >
        comment 1
      </p>
      <p
        class="sc-hKgILt fLjEpC"
      >
        comment 2
      </p>
    </section>
  </section>
  <section
    class="sc-eCssSg foUHDZ"
  >
    <h1
      class="sc-jSgupP dhcVWK"
    >
      QA
    </h1>
    <section
      class="sc-bdfBwQ bCQgUO"
    >
      <h2
        class="sc-gsTCUz gzfVJk"
      >
        mock task title
      </h2>
      <p
        class="sc-dlfnbm bglaea"
      >
        A mock description
      </p>
      <p
        class="sc-hKgILt fLjEpC"
      >
        comment 1
      </p>
      <p
        class="sc-hKgILt fLjEpC"
      >
        comment 2
      </p>
    </section>
  </section>
  <section
    class="sc-eCssSg foUHDZ"
  >
    <h1
      class="sc-jSgupP dhcVWK"
    >
      Done
    </h1>
    <section
      class="sc-bdfBwQ bCQgUO"
    >
      <h2
        class="sc-gsTCUz gzfVJk"
      >
        mock task title
      </h2>
      <p
        class="sc-dlfnbm bglaea"
      >
        A mock description
      </p>
      <p
        class="sc-hKgILt fLjEpC"
      >
        comment 1
      </p>
      <p
        class="sc-hKgILt fLjEpC"
      >
        comment 2
      </p>
    </section>
  </section>
</section>
`;
